src/tables/helpers/apply-value-getters.js

```diff
@@ -0,0 +1,16 @@
+export default function applyValueGetters(columns, data) {
+  let annotatedData = data
+
+  columns.forEach(({ name, valueGetter }) => {
+    if (valueGetter) {
+      annotatedData = data.map((datum) => {
+        return {
+          ...datum,
```

Comment:
This will mess with the shape of the object when it's passed to a custom row or cell component. It would be better to just call `valueGetter()` twice, when sorting and when rendering.

Reply:
Updated, as discussed